src/main/java/hudson/plugins/release/dashboard/RecentReleasesPortlet.java

```diff
@@ -119,7 +119,7 @@ private void rss(StaplerRequest req, StaplerResponse rsp, String suffix, RunList
 		
 		@Extension
 		public static DescriptorImpl newInstance() {
-			if (Hudson.getInstance().getPlugin("dashboard-view") != null) {
+			if (Jenkins.getActiveInstance().getPlugin("dashboard-view") != null) {
```

Comment:
Typically better done with `optional` extensions, but I am not familiar with the specifics here.